sn_node/src/node/bootstrap/relocate.rs

```diff
@@ -301,7 +301,7 @@ mod tests {
                 }
                 if !matches!(
                     msg,
-                    OutgoingMsg::System(SystemMsg::JoinAsRelocatedRequest(_))
+                    OutgoingMsg::Node2Node(NodeMsg::JoinAsRelocatedRequest(_))
```

Comment:
Looks like you may have missed the tests. Try verifying with `cargo clippy --all-targets`.
```suggestion
                    OutgoingMsg::Node(NodeMsg::JoinAsRelocatedRequest(_))
```